### PR TITLE
docs/nia: update status api with new language and calculations

### PR DESCRIPTION
**Pull request by lornasong**
- Renamed: 'undetermined' => 'unknown', 'healthy' => 'successful',
 'degraded' => 'errored'
 - Updated language for task status to determine status based on most recent
 event
 - Updated language on overall status to summarize count of statuses

[Task preview link](https://deploy-preview-9406--consul-docs-preview.netlify.app/docs/nia/tasks#status-information)
[Api preview link](https://deploy-preview-9406--consul-docs-preview.netlify.app/docs/nia/api#status)

### CODE REVIEW

File: website/pages/docs/nia/tasks.mdx [hunk 1]
```diff
@@ -88,7 +88,7 @@ For complete information on the event structure, see [events in our API document
 
 ### Task Status
 
-Each time a task runs to update network infrastructure, event data is stored for that run. Currently, only the 5 most recent events are stored for each task. The stored events are used to determine the task status. For example, if a task has 5 events stored and 3 are success and 2 are not, then the task's health status is "degraded".
+Each time a task runs to update network infrastructure, event data is stored for that run. Currently, only the 5 most recent events are stored for each task. The stored events are used to determine the task status. For example, if a task has 5 events stored and the most recent is not successful but all previous ones are, then the task's health status is "degraded".
```
Comment (findkim) [inline website/pages/docs/nia/tasks.mdx:91]: This reads like it still needs to be updated, mix of old and new logic.

File: website/pages/docs/nia/api.mdx [hunk 2]
```diff
@@ -83,10 +79,10 @@ Response:
 This endpoint returns the individual task status information for a single specified task or for all tasks.
 
 Task health status value is determined by the success or failure of all stored [event data](/docs/nia/tasks#event) on the process of updating network infrastructure for a task. Currently only the 5 most recent events are stored per task.
- - Healthy: All stored events of the task are successful.
- - Degraded: More than half of the stored events are successful _or_ less than half of the stored events are successful but the most recent event is successful.
- - Critical: Half or less of the stored events are successful and the most recent event is not successful.
- - Undetermined: No event data is stored for the task.
+ - Successful: The most recent stored event is successful.
+ - Errored: The most recent stored event is not successful but all previous stored events are successful.
+ - Critical: The most recent stored event is not successful and more than one previous stored event is also not successful.
```
[the remark this answers is not in the document]
Comment (lornasong) [reply website/pages/docs/nia/api.mdx:84]: thanks for catching this!

File: website/pages/docs/nia/tasks.mdx (hunk 1 above)
Comment (lornasong) [reply website/pages/docs/nia/tasks.mdx:91]: Ack, a 'degraded' made it through. Thanks! What do you think about below?

> Each time a task runs to update network infrastructure, event data is stored for that run. Currently, only the 5 most recent events are stored for each task, and these stored events are used to determine task status. For example, if the most recent stored event is not successful but the others are, then the task's health status is "errored"

Comment (findkim) [reply website/pages/docs/nia/tasks.mdx:91]: I would suggest to remove the "Currently, only the 5" and just go with a statement that "5 most recent events are stored for each task, ..."

The new changes look good 👍

Comment (lornasong) [reply website/pages/docs/nia/tasks.mdx:91]: Note: seeing another 'degraded' a little lower down line 97

**Review by findkim — APPROVED** [review]
🚢 🚢 🚢

**Pull request merged**